tensorflow/python/keras/losses.py

```diff
@@ -810,6 +815,8 @@ def logloss(y_true, y_pred):
   return K.mean(-losses, axis=-1)
 
 
+@keras_export('keras.losses.huber',
+              'keras.losses.huber_loss')
 def huber_loss(y_true, y_pred, delta=1.0):
```

Comment:
Can we update the name to huber and just export `keras.losses.huber`?

Reply:
Done 😄